modeler.py

```diff
@@ -14,17 +16,21 @@ def __call__(self, state, action):
         Args:
           state (object): Current world state to start from.
           action (object): Action to take in current world state.
+
         None action is not allowed.
 
         Returns:
           (object): Next state after taking the given action in the given state.
+
         If action is not allowed, returns None.
```

Comment:
To nie powinno być oddzielone eterem, tylko wcięte, że jest na równoważni z tamtym, ale pod punktem "Return:". Teraz jest jakieś opuszczone, nie wiadomo do czego należy.